docs/api-plugins-analytics-plugin.mdx

```diff
@@ -0,0 +1,68 @@
+---
+id: api-plugins-analytics-plugin
+slug: /search-ui/api/core/plugins/analytics-plugin
+title: Analytics Plugin
+date: 2023-01-05
+tags: [""]
+---
+
+The Analytics Plugin is a plugin that provides a way to send analytics events to the Behavioral Analytics Product. To use it, you must follow the instructions on how to set up the Analytics Product.
+
+## Installation
+
+```bash
+yarn add @elastic/search-ui-analytics-plugin
+## OR
+npm install @elastic/search-ui-analytics-plugin
+```
+
+## Basic Usage
+
+Within your Search UI configuration, you can add the Analytics Plugin like so:
```

Comment:
Is there a missing example here after "like so:"?

Reply:
the example is below doesn't follow immediately after. Ive updated it to be immediately after :)